Doc/reference/expressions.rst

```diff
@@ -1829,7 +1828,8 @@ precedence and have a left-to-right chaining feature as described in the
 | ``x[index]``, ``x[index:index]``,             | Subscription, slicing,              |
 | ``x(arguments...)``, ``x.attribute``          | call, attribute reference           |
 +-----------------------------------------------+-------------------------------------+
-| ``(expressions...)``,                         | Binding or tuple display,           |
+| ``(expressions...)``,                         | Binding or parenthesized            |
+|                                               | expressions,                        |
```

Comment:
Doesn’t this split the round bracket syntax into a separate paragraph from the other brackets?

Reply:
Yes, I think you are right!